en/orm/entities.rst

```diff
@@ -389,13 +389,17 @@ protect itself against mass assignment::
 Modifying the Guarded Fields at Runtime
 ---------------------------------------
 
-You can modify the list of guarded fields at runtime using the ``accessible``
+You can modify the list of guarded fields at runtime using the ``setAccess``
```

Comment:
IMO we should add the () brackets for methods.

Reply:
i agree, i will fix that too.